llvm/CODE_OWNERS.TXT

```diff
@@ -263,3 +263,7 @@ D: C-SKY backend (lib/Target/CSKY/*)
 N: Ilia Diachkov
 E: ilia.diachkov@gmail.com
 D: SPIR-V backend (lib/Target/SPIRV/*)
+
+N: Christopher Apple, David Trevelyan
+E: cja-private@pm.me, realtime.sanitizer@gmail.com
```

Comment:
It's better to use an email address for contribution instead of `realtime.sanitizer`, which might forward to you, but are not associated with any commits.

Reply:
Fixed in the latest version!